guidesmiths/systemic-aws-s3
# Adding linter

### What’s the focus of this PR
Hello.

This PR adds Eslint, its configuration and scripts to validate style and quality items in the project

I took a look at `semistandard`, but I think `eslint` is easier to configure, has easier error viewing, and has community support.

### How to review this PR

* Check file changes
* Run `npm install`
* Run the command `npm run lint`, which will show improvement points per file.

### Related work items
- #16 

### Before submitting this PR, I made sure:
- [ ] I've added tests
